enhancements/two-node-fencing/tnf.md

```diff
@@ -567,23 +660,26 @@ Additionally, it would be good to have workload-specific testing once those are
 
 ### Dev Preview -> Tech Preview
 
-- Ability to install a two-node cluster using assisted installer (via ACM) and agent-based installer
+- Ability to install a two-node cluster using the core installer via a bootstrap node
+- Ability to install a two-node cluster using assisted installer (via ACM)
 - End user documentation, relative API stability
-- Sufficient test coverage (see test plan above)
+- CI that shows verifies that nightlies can be installed and pass most conformance tests
+- QE verification that the 2-node clusters can handle graceful and ungraceful failures
 
 ### Tech Preview -> GA
 
+- Telco input is gathered, documented, and addressed based on feedback
+- OLM allows for operator owners to opt-in or out of 2-node topology support
+- Ability to install a 2-node cluster using the agent-based installer
 - Working upgrades
-- Upgrade tests
-- Available by default
+- Available by default (no longer in the `CustomNoUpgrade` feature gate)
```

Comment:
nit: gate -> set

And I think this actually means that the `feature` should not be used anymore and should be to be removed at all, because IIUC you can't enable / disable a single feature without using the `CustomNoUpgrade` feature set. A feature is either enabled / disabled in the default feature set, or in the tech preview feature set (both immutable), or manually with the `CustomNoUpgrade` feature set. I don't think there is a way for "enable this feature on the fly and keep the cluster upgradable" :thinking: 
That also means that the feature gate should not be used by anything, it's 
purely for preventing upgrades, right?

So I guess this can be "Working upgrades, feature gate not used anymore"?

Reply:
while the feature gate does prevent upgrades, I would consider it a different objective than working upgrades - which implies there is a test-provable paper trail that upgrades are functioning as expected.

I've kept it at its, but basically just reworded the line to clarify that the feature gate no longer blocks access to the feature.

Reply:
wfm 👍🏼